playbook/app/pb_kits/playbook/pb_advanced_table/docs/_advanced_table_default.html.erb

```diff
@@ -0,0 +1,33 @@
+<% require './app/helpers/advanced_table_example_mock_data.rb' %>
```

Comment:
I'd love to expose this like we did for react

Reply:
Sure, we can chat about it!